docs/source/cuda_environment_variables.rst

```diff
@@ -0,0 +1,41 @@
+.. _cuda_environment_variables:
+
+CUDA Environment Variables
+==========================
+.. list-table::
+  :header-rows: 1
+
+  * - Variable
```

Comment:
I think we should link to https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#env-vars here for people looking for more details.

Reply:
great idea!